src/UserService.Models.Dto/Requests/User/CreateUserRequest.cs

```diff
@@ -13,12 +13,12 @@ public record CreateUserRequest
         public string LastName { get; set; }
         public string MiddleName { get; set; }
         public UserGender Gender { get; set; }
-        public string DateOfBirth { get; set; }
+        public DateTime DateOfBirth { get; set; }
```

Comment:
They should be 'DateTime?' because it is valueType and it is not necessary according to specification

Reply:
StartWorkingAt too

Reply:
May be time we not necessary?